src/library/autodj/dlgautodj.cpp

```diff
@@ -372,6 +380,22 @@ void DlgAutoDJ::updateSelectionInfo() {
     }
 }
 
+void DlgAutoDJ::updateInfo() {
+    double duration = 0.0;
+
+    int rows = m_pAutoDJTableModel->rowCount();
+    for (int row = 0; row < rows; ++row) {
+        TrackPointer pTrack = m_pAutoDJTableModel->getTrack(m_pAutoDJTableModel->index(row, 0));
+        duration += pTrack->getDuration();
+    }
+
+    QString label;
+    label.append(mixxx::DurationBase::formatTime(duration));
```

Comment:
This needs to communicate that the shown time is an estimate. Users making plans based on this will be mislead if they think this is accurate!

Reply:
maybe a prefix like "Sum:" clarifies the overlaps/transitions are not considered.
Anyway, such info should be in the tooltip.

Reply:
I disagree that it should be hidden in the tooltip. Also "sum" is unclear. "Estimate" would convey that this is not reliable.